lib/tasks/fix_missing_notification_settings.rake

```diff
@@ -0,0 +1,44 @@
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) 2012-2022 the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+namespace :migrations do
+  desc "Fix the missing notification settings"
+  task fix_missing_notification_settings: :environment do
+    ActiveRecord::Base.connection.execute <<~SQL.squish
+      INSERT INTO
+        notification_settings
+        (user_id, watched, involved, mentioned)
```

Comment:
I do not know if that pull request is still relevant. It probably is.

As default value for `watched`, `involved`, and `mentioned` are already `true`, they can be removed. That will make the code more robust as `involved` does not exist anymore and has been replaced by `assignee` and `responsible`.